content/2021/2021-06/2021-06-28-pulsar-weekly.md

```diff
@@ -0,0 +1,241 @@
+
+    https://github.com/apache/pulsar/pull/11035 ([@hangc0276](https://github.com/hangc0276))
+
+- [Tests] Fix the `AdminApiOffloadTest` Flaky test.
+
+    https://github.com/apache/pulsar/pull/11028 ([@hangc0276](https://github.com/hangc0276))
+
+- [Tests] Fix the `ResourceGroupConfigListenerTest` Flaky test.
+
+    https://github.com/apache/pulsar/pull/11048 ([@lhotari](https://github.com/lhotari))
+
+- [Python] Fix the error in importing the Pulsar 2.8.0 Python client when AvroSchema is not used.
+
+    https://github.com/apache/pulsar/pull/11034 ([@merlimat](https://github.com/merlimat))
+
+- [Tests] Fix the `AdminApiTest` Flaky test.
+
+    https://github.com/apache/pulsar/pull/11059 ([@mattisonchao](https://github.com/mattisonchao))
+
+- [Broker] Make the `PulsarClusterMetadataTeardown` deletions idempotent.
+
+    https://github.com/apache/pulsar/pull/11042 ([@merlimat](https://github.com/merlimat))
+
+- [C++] Fix the issue that the C++ source code is compiled unsuccessfully for Windows 32-bit build.
+
+    https://github.com/apache/pulsar/pull/11082 ([@BewareMyPower](https://github.com/BewareMyPower))
+
+- [Tests] Fix the `testTransactionMetaStoreAssignAndFailover` Flaky test.
+
+    https://github.com/apache/pulsar/pull/11073 ([@mattisonchao](https://github.com/mattisonchao))
+
+- [Broker] Fix the incorrectly-used loggers.
+
+    https://github.com/apache/pulsar/pull/11047 ([@mattisonchao](https://github.com/mattisonchao))
+
+- [Broker] Add the authoritative flag for the topic-level policy to avoid the redirect loop.
+
+    https://github.com/apache/pulsar/pull/11051 ([@codelipenghui](https://github.com/codelipenghui))
+
+- [Test] Fix the `testDuplicateConcurrentSubscribeCommand` Flaky test.
+
+    https://github.com/apache/pulsar/pull/11089 ([@hangc0276](https://github.com/hangc0276))
+
+- [C++] Fix the issue that the `KeySharedMeta` is not set as a subscribe command.
+
+    https://github.com/apache/pulsar/pull/11088 ([@k2la](https://github.com/k2la))
+
+- [Topic Policy] Fix the issue that the topic-level retention policy does not work.
+
+    https://github.com/apache/pulsar/pull/11021 ([@hangc0276](https://github.com/hangc0276))
+
+- [Functions] Fix the build failure caused by the SpotBugs.
+
+    https://github.com/apache/pulsar/pull/10792 ([@Jason918](https://github.com/Jason918))
+
+- [Build] Fix the compilation error caused by the the number of incorrect parameters.
+
+    https://github.com/apache/pulsar/pull/11104 ([@mattisonchao](https://github.com/mattisonchao))
+
+- [Broker] Fix the issue that the response filter processes asynchronous responses incorrectly.
+
+    https://github.com/apache/pulsar/pull/11052 ([@gaoran10](https://github.com/gaoran10))
+
+- [Topic Policy] Fix the `SystemTopicBasedTopicPoliciesService` race condition.
+
+    https://github.com/apache/pulsar/pull/11097 ([@codelipenghui](https://github.com/codelipenghui))
+
+### Event / News
+
+- [TGIP] Weekly live stream about Pulsar and its ecosystem.
```

Comment:
There will be big changes to TGIP name and episodes soon, you can add it below other impactful events in the future.
For example, you can introduce Pulsar 101 first and then TGIP.